std/node/util.ts

```diff
@@ -4,13 +4,33 @@ import * as types from "./_util/_util_types.ts";
 
 export { types };
 
+const DEFAULT_INSPECT_OPTIONS = {
+  showHidden: false,
+  depth: 2,
+  colors: false,
+  customInspect: true,
+  showProxy: false,
+  maxArrayLength: 100,
+  maxStringLength: Infinity,
+  breakLength: 80,
+  compact: 3,
+  sorted: false,
+  getters: false,
+};
+
+inspect.defaultOptions = DEFAULT_INSPECT_OPTIONS;
+inspect.custom = Deno.customInspect;
+
+// TODO: make it in-line with Node's implementation
+// Ref: https://nodejs.org/dist/latest-v14.x/docs/api/util.html#util_util_inspect_object_options
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function inspect(object: unknown, ...opts: any): string {
+  opts = { ...DEFAULT_INSPECT_OPTIONS, ...opts };
   return Deno.inspect(object, {
-    depth: opts.depth ?? 4,
-    iterableLimit: opts.iterableLimit ?? 100,
-    compact: !!(opts.compact ?? true),
-    sorted: !!(opts.sorted ?? false),
+    depth: opts.depth,
+    iterableLimit: opts.maxArrayLength,
+    compact: !!opts.compact,
+    sorted: !!opts.sorted,
```

Comment:
I forgot, `showProxy` support has landed in the meanwhile. You can add it here now.
```ts
    showProxy: opts.showProxy,
```

Reply:
This is not resolved yet, just append the line above @bartlomieju.